lib/src/main/java/com/jesusm/holocircleseekbar/lib/HoloCircleSeekBar.java

```diff
@@ -313,22 +328,30 @@ protected void onDraw(Canvas canvas) {
 
         canvas.translate(mTranslationOffset, mTranslationOffset);
 
+        // Draw the pointer's "halo"
+        canvas.drawCircle(pointerPosition[0], pointerPosition[1],
+                mColorWheelStrokeWidth * 1.70f, mPointerHaloPaint2);
+
         // Draw the color wheel.
         canvas.drawArc(mColorWheelRectangle, start_arc + 270, end_wheel
                 - (start_arc), false, mColorWheelPaint);
 
         canvas.drawArc(mColorWheelRectangle, start_arc + 270,
                 (arc_finish_radians) > (end_wheel) ? end_wheel - (start_arc)
+                        : arc_finish_radians - start_arc, false, mArcColor2);
+
+        canvas.drawArc(mColorWheelRectangle, start_arc + 270,
+                (arc_finish_radians) > (end_wheel) ? end_wheel - (start_arc)
                         : arc_finish_radians - start_arc, false, mArcColor);
 
         // Draw the pointer's "halo"
         canvas.drawCircle(pointerPosition[0], pointerPosition[1],
-                mPointerRadius, mPointerHaloPaint);
+                mColorWheelStrokeWidth * 1.40f, mPointerHaloPaint);
```

Comment:
I don't see why it should use `mColorWheelStrokeWidth` rather than the previous `mPointerRadius` attribute. `mPointerRadius` defines how big the circle should be, and the one you propose here handles a different part of the widget. In case you want to use have the same dimension for both parts, you can just use same dimension for both.